--- a/exrenvmap/main.cpp
+++ b/exrenvmap/main.cpp
@@ -46,6 +46,8 @@
 #include <exception>
 #include <stdlib.h>
 
+#include <cstring>
+
 using namespace Imf;
 using namespace std;
 
--- a/exrmaketiled/main.cpp
+++ b/exrmaketiled/main.cpp
@@ -47,6 +47,8 @@
 #include <string>
 #include <stdlib.h>
 
+#include <cstring>
+
 using namespace Imf;
 using namespace std;
 
diff -aurN openexr-1.7.0.orig/exrenvmap/blurImage.cpp openexr-1.7.0/exrenvmap/blurImage.cpp
--- openexr-1.7.0.orig/exrenvmap/blurImage.cpp	2011-02-06 14:19:53.000000000 +0100
+++ openexr-1.7.0/exrenvmap/blurImage.cpp	2011-02-06 14:20:24.000000000 +0100
@@ -45,6 +45,7 @@
 #include "Iex.h"
 #include <iostream>
 #include <algorithm>
+#include <cstring>

 using namespace std;
 using namespace Imf;
